Sources/AuthFoundation/Network/APIClient.swift

```diff
@@ -91,16 +123,25 @@ extension APIClient {
         defaultJSONDecoder.userInfo = [:]
         return try? defaultJSONDecoder.decode(OktaAPIError.self, from: data)
     }
-
+    
     public func willSend(request: inout URLRequest) {}
     
     public func didSend(request: URLRequest, received error: APIClientError) {}
-
+    
     public func didSend<T>(request: URLRequest, received response: APIResponse<T>) {}
-
+    
     public func send<T>(_ request: URLRequest, parsing context: APIParsingContext? = nil, completion: @escaping (Result<APIResponse<T>, APIClientError>) -> Void) {
+        send(request, parsing: context, state: nil, completion: completion)
+    }
+    
+    public func shouldRetry(request: URLRequest, rateLimit: APIRateLimit) -> APIRetry { .default }
+    
+    private func send<T>(_ request: URLRequest,
+                         parsing context: APIParsingContext? = nil,
+                         state: APIRetry.State?,
+                         queue: DispatchQueue = .global(),
+                         completion: @escaping (Result<APIResponse<T>, APIClientError>) -> Void) {
         var urlRequest = request
-        
         willSend(request: &urlRequest)
         session.dataTaskWithRequest(urlRequest) { data, response, httpError in
```

Comment:
SwiftLint is still complaining about this block. Could you see about either refactoring some of it to be smaller, or we can consider increasing the closure_body_length limit.

Reply:
Good callout. I've updated this.